server-base/src/main/java/org/apache/kylin/rest/response/SQLResponse.java

```diff
@@ -73,9 +73,13 @@
     protected boolean queryPushDown = false;
 
     protected byte[] queryStatistics;
-    
+
     protected String traceUrl = null;
 
+    protected String signature;
+
+    protected long dummyTime = -1L;
```

Comment:
better to have more comments about these 2 fields: their usage and why they're json-ignored.

Reply:
agree